staging/src/k8s.io/kubectl/pkg/util/podutils/podutils.go

```diff
@@ -186,3 +186,48 @@ func maxContainerRestarts(pod *corev1.Pod) int {
 	}
 	return maxRestarts
 }
+
```

Comment:
Sounds good - added in 88d19f4b5aa505694443c724c6c0aff492af8923

I also noticed that my `VisitContainers` was taking a `*pod` instead of a `*podspec` and I've updagted that in 9980209146c0c5059b26f6fcccb765d7d263a8b9 so that it's now consistent with what's in k/k

Reply:
Thanks for noticing the inconsistency!